Add steps to obtain elevator access to internal onboarding ticket (#12)
ccao-data/people: .github/ISSUE_TEMPLATE/onboard-returning.md

- [ ] Verify that you have access to your existing CCAO email.
- [ ] Verify that you have VPN access.

> - [ ] Verify that you have an ID badge with security access to 9th floor.

Reviewer: ```suggestion
- [ ] Verify that your County employee ID badge grants access to 9th floor.
```
